README.md

```diff
@@ -8,17 +8,20 @@
 
 ## Prerequisites
 
+Pre-built `cofidectl` binaries may be downloaded from the project's [GitHub releases](https://github.com/cofide/cofidectl/releases) page.
```

Comment:
I find it confusing to read this in "prerequisites". Surely, these are prerequisites to building so I would see this in its own section ("binary builds"? "quick install"?). We can rename prerequisites to "Build prerequisites" to be very clear on the purpose (but that's less important).